src/util/cache-utils.ts

```diff
@@ -0,0 +1,33 @@
+import * as ghCore from "@actions/core";
+import { URL } from "url";
+import { Inputs } from "../generated/inputs-outputs";
+
+const SKIP_CACHE_ENVVAR = "CLI_INSTALLER_SKIP_CACHE";
+const skipCache = ghCore.getInput(Inputs.SKIP_CACHE);
+
+export function shouldUseCache(operation: string): boolean {
```

Comment:
should you narrow the type of `operation` to `"check" | "upload"` ?

Reply:
i'm not certain you need to pass this anyway. if you just log `skipping cache` that is fine.

Reply:
Okay, will remove those